packages/main/src/TableSelection.ts

```diff
@@ -204,6 +216,180 @@ class TableSelection extends UI5Element implements ITableFeature {
 		this._table.headerRow[0]._invalidate++;
 		this._table.rows.forEach(row => row._invalidate++);
 	}
+
+	_onkeydown(e: KeyboardEvent) {
+		if (!this.isMultiSelect() || !this._table || !e.shiftKey) {
+			return;
+		}
+
+		const focusedElement = getActiveElement(); // Assumption: The focused element is always the "next" row after navigation.
+
+		if (!isInstanceOfTableRow(focusedElement) && !this._rangeSelection?.isMouse) {
```

Comment:
f I select the last row and shift arrow down and then I am on the growing button should then the range selection stop?